blockly/blocks/base.js

```diff
@@ -196,3 +196,56 @@ Blockly.Blocks['serial_print'] = {
     this.setTooltip('Prints data to the console/serial port as human-readable ASCII text.');
   }
 };
+
+Blockly.Blocks['lcd_setup'] = {
+  init:function(){
```

Comment:
I completely agree with this. It obviously slipped here and I'll make sure next time to follow these more closely!